diff --git a/pyproject.toml b/pyproject.toml
index 41c43e4..8de4c63 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,13 +26,11 @@ test-command = 'python -c "import vroom"'
 build = "cp*"
 skip = "*musllinux*"
 archs = "native"
-manylinux-x86_64-image = "quay.io/pypa/manylinux_2_28_x86_64"
 
 [tool.cibuildwheel.linux]
 before-all = """
-dnf update -y
-dnf module enable -y mariadb-devel
-dnf install -y openssl-devel asio-devel
+yum install -y epel-release
+yum install -y openssl-devel asio-devel
 """
 
 [[tool.cibuildwheel.overrides]]
diff --git a/setup.cfg b/setup.cfg
index f5401e6..6959462 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 version = 0.1.0
-name = pyvroom
+name = pyvroomwheels
 description = Vehicle routing open-source optimization machine (VROOM)
 long_description = file: README.rst
 long_description_content_type = text/x-rst
diff --git a/setup.py b/setup.py
index 8d838ef..f560dcd 100644
--- a/setup.py
+++ b/setup.py
@@ -80,6 +80,6 @@ setup(
     ext_modules=ext_modules,
     ext_package="vroom",
     include_dirs=include_dirs,
-    use_scm_version=True,
+    version="1.13.5",
     entry_points={"console_scripts": ["vroom=vroom:main"]},
 )
